lib/utils/utils.ts

```diff
@@ -134,3 +135,14 @@ export const groupBy = (arr: object[], keyGetter: (item: any) => string | number
 export const ms = (): number => {
   return Date.now();
 };
+
+export const getEnumNumbers = (enumObj: any) => {
```

Comment:
Is this function used somewhere?

Reply:
Oops nope, I was using it at one point for putting restrictions on the eligible network clients in the database, but abandoned that approach. Thanks.